include/toolchain/gcc.h

```diff
@@ -113,6 +113,11 @@ do {                                                                    \
 
 #define __in_section_unique(seg) ___in_section(seg, __FILE__, __COUNTER__)
 
+/* Using this places a function in RAM instead of FLASH.
+ */
+#define __ramfunc	__attribute__((noinline))			\
```

Comment:
We cannot use identifiers starting with double undescores. cc: @ceolin

I'd recommend to follow standard preprocessor macro convention and call it something like RAMFUNC, but seems we already have things like `_app_dmem(id)` and `_app_bmem(id)` ;-/ , so `_ramfunc` would do...

Reply:
Or maybe something like _GENERIC_SECTION()?

Reply:
There has some __attribute__ marco starting with double underscores, such as:
__weak, __used, __unused, __packed, __aligned, So I prefer using __ramfunc.